plugins/memblaze/memblaze-nvme.c

```diff
@@ -402,11 +414,12 @@ int parse_params(char *str, int number, ...)
 	return 0;
 }
 
-static int mb_get_additional_smart_log(int argc, char **argv, struct command *cmd, struct plugin *plugin)
+static int mb_get_additional_smart_log(
+	int argc, char **argv,
+	struct command *cmd, struct plugin *plugin)
```

Comment:
Indent these two lines with double tap, so the arguments do not align with the single tab indented variable declearations.